forte/data/data_pack_dataset.py

```diff
@@ -0,0 +1,149 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Iterator, Type, Optional, List, Tuple, Union, Any
+
+import torch
+from texar.torch import HParams
+from texar.torch.data import IterDataSource, DatasetBase, Batch
+
+from forte.data.converter.converter import Converter
+from forte.data.converter.feature import Feature
+from forte.data.data_pack import DataPack
+from forte.data.extractor.base_extractor import BaseExtractor
+from forte.data.ontology.core import EntryType
+from forte.data.ontology.top import Annotation
+from forte.data.readers.base_reader import PackReader
+from forte.data.types import DataRequest
+
```

Comment:
I think we need docstrings for the functions here.

Reply:
Done